src/main/java/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -465,7 +462,7 @@ class TypeSpec private constructor(builder: TypeSpec.Builder) {
     }
 
     private fun ensureCanHaveSuperclass() {
-      check(kind.isSimpleClass || kind is Object) {
+      check(kind.isSimpleClass || kind.isEnum || kind is Object) {
         "only classes can have super classes, not $kind"
```

Comment:
This message is starting to lose its accuracy.

Reply:
whoops, saw that earlier and forgot to fix. brb.